src/main/java/net/fabricmc/api/EnvType.java

```diff
@@ -16,7 +16,35 @@
 
 package net.fabricmc.api;
 
+/**
+ * Represents a type of environment.
+ *
+ * <p>A type of environment is a jar file in a <i>Minecraft</i> version's json file's {@code download}
+ * subsection, including the {@code client.jar} and the {@code server.jar}.</p>
+ *
+ * @see Environment
+ * @see EnvironmentInterface
+ * @see <a href="https://launchermeta.mojang.com/mc/game/version_manifest.json">A list of <i>Minecraft</i>
+ * version's json files</a>
+ */
```

Comment:
this doesnt seem great, no idea why they need the launcher meta json here.

Reply:
Just a reference for people for json so they can actually see the client/server sections in downloads section.

Reply:
Removed the launchermeta link.